libs/spoke-codegen/src/graphql/campaign-stats.graphql

```diff
@@ -54,6 +59,7 @@ query getOrganizationData($organizationId: String!) {
         createdAt
       }
     }
+    messagingServicesCount
```

Comment:
Question: Why add a new query instead of just a `messagingServices.count` type thing?

Reply:
I think I went in this direction to practice with sql 😆 

Updated to use pageInfo.totalCount.